modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/TagEngageWorkflowOperationHandler.java

```diff
@@ -134,7 +134,8 @@ public WorkflowOperationResult start(WorkflowInstance instance, JobContext conte
     try {
       mediaPackageForSearch = searchService.get(mediaPackageId);
     } catch (NotFoundException | UnauthorizedException e) {
-      throw new WorkflowOperationException(e);
+      logger.info("No tag engage workflow found for {}", mediaPackageId);
```

Comment:
That logging message isn't quite correct: I'd write something like "Media package {} isn't published to Engage, skipping." Because the workflow does exist, and so does the media package, but not in the SearchService/Engage.